app/src/main/java/com/simformsolutions/oneloop/ui/user/navigation/UserNavigation.kt

```diff
@@ -31,5 +38,45 @@ fun NavGraphBuilder.userNavigation() {
                     .imePadding()
             )
         }
+
+        composable<SignupUserInfo> {
+            SignupUserInfoRoute(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .imePadding()
+            )
+        }
+
+        composable<SignupPassword> {
+            SignupPasswordRoute(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .imePadding()
+            )
+        }
+
+        composable<SignupPhoneVerification> {
+            SignupPhoneVerificationRoute(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .imePadding()
+            )
+        }
+
+        composable<SignupOtpVerification> {
+            SignupOtpVerificationRoute(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .imePadding()
```

Comment:
Do you need to apply `imePadding` here? Because it's already applied to the `Column` that is inside the scaffold of this screen.

Reply:
Done